rsmtool/utils/commandline.py

```diff
@@ -783,6 +785,11 @@ def interact(self):
         sys.stderr.write(" - you may still need to edit the generated configuration\n")
         sys.stderr.write("\n")
 
+        sys.stderr.write("IMPORTANT: If you have subgroups and didn't specify the '-g' "
+                         "option, exit now (ctrl-d) and re-run!\n")
```

Comment:
```suggestion
        if not self.use_subgroups:
                sys.stderr.write("IMPORTANT: If you have subgroups and didn't specify the '-g' "
                                           "option, exit now (ctrl-d) and re-run!\n")
```

I just tried -ig flag and was very confused t see this message.

Reply:
Good catch!